libs/clients/judicial-administration/src/lib/providers.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Configuration,
+  AuthenticateApi,
+  CreateCaseApi,
+  CreateCustodyCaseApi,
+  CreateDocumentApi,
+  CreateEmailApi,
+  CreateThingbokApi,
+  GetCaseSubtypesApi,
+  LookupDataGDNJCasesApi,
+  ParticipantsApi,
+  SearchBankruptcyHistoryApi,
+} from '../../gen/fetch'
+import { ApiConfig } from './api.config'
+
+export const exportedApis = [
+  AuthenticateApi,
+  CreateCaseApi,
+  CreateCustodyCaseApi,
+  CreateDocumentApi,
+  CreateEmailApi,
+  CreateThingbokApi,
+  GetCaseSubtypesApi,
+  LookupDataGDNJCasesApi,
+  ParticipantsApi,
+  SearchBankruptcyHistoryApi,
```

Comment:
Are you only using the `AuthenticateApi` and `SearchBankruptcyHistoryApi`?
I would only add the used apis in exported apis, and add the other on demand.